Userland/DevTools/HackStudio/Language.cpp

```diff
@@ -37,4 +37,27 @@ Language language_from_name(const String& name)
     return Language::Unknown;
 }
 
+String language_name_from_file_extension(const String& extension)
+{
+    VERIFY(!extension.starts_with("."));
+    if (extension == "cpp" || extension == "h")
+        return "Cpp";
```

Comment:
Why `Cpp` instead of `C++`?

Reply:
I matched how it was referenced in the `Language::language_from_name` function to keep it consistent, but I do think that `C++` looks better. I can change it.

Reply:
I wonder why it's `Cpp` in `LanguageServer`. I don't think it's shown to the user though, so it probably doesn't matter.